### ISSUE TITLE
Shows no contents in bags

### ISSUE THREAD

**Issue by nymalt** [Cidan/BetterBags#664]
### Addon Version

v0.3.7

### World of Warcraft Version

Classic Era (And SoD/Hardcore)

### Describe the bug

After latest update the contents of bags are blank - just showing nothing in the bags

### Steps to reproduce

Just open bag plainly

**Comment by Jonnnyki** [comment]
Same issue for Cata Classic with release version of v0.3.7 and alpha version of v0.3.6. Downgrading to release v0.3.6 seems to be  an easy temporary solution till the problem is fixed.

**Comment by ManuelPhelouzat** [comment]
Confirmed on Cata Classic, reverted to v0.3.5 to avoid the bag not updating bug.

**Comment by sunderops** [comment]
Confirmed same issue with SoD. Reverting to previous version for now.

**Comment by Eldarock** [comment]
Same issue in retail with 0.3.8

**Comment by AMaribo** [comment]
> Same issue in retail with 0.3.8

Are you receiving any Lua errors?

**Comment by Eldarock** [comment]
`6x BetterBags/core/async.lua:49: Blizzard_ObjectAPI/Mainline/Item.lua:121: bad argument #1 to 'GetItemGUID' (Usage: local itemGUID = C_Item.GetItemGUID(itemLocation))
[string "=[C]"]: in function `error'
[string "@BetterBags/core/async.lua"]:49: in function <BetterBags/core/async.lua:46>

Locals:
(*temporary) = "Blizzard_ObjectAPI/Mainline/Item.lua:121: bad argument #1 to 'GetItemGUID' (Usage: local itemGUID = C_Item.GetItemGUID(itemLocation))"`

tested with 0.3.8 just after the weekly reset.
French client.

**Comment by davethemage** [comment]
I am getting a similar issue in retail(0.3.9). My bag has nothing in it.
```
4x BetterBags/core/async.lua:49: Blizzard_ObjectAPI/Mainline/Item.lua:121: bad argument #1 to 'GetItemGUID' (Usage: local itemGUID = C_Item.GetItemGUID(itemLocation))
[string "=[C]"]: in function `error'
[string "@BetterBags/core/async.lua"]:49: in function `worker'
[string "@BetterBags/core/async.lua"]:65: in function `DoWithDelay'
[string "@BetterBags/core/async.lua"]:74: in function `Do'
[string "@BetterBags/core/async.lua"]:102: in function `Batch'
[string "@BetterBags/data/loader.lua"]:111: in function `Load'
[string "@BetterBags/data/items.lua"]:799: in function `ProcessContainer'
[string "@BetterBags/data/items.lua"]:515: in function `RefreshBackpack'
[string "@BetterBags/data/refresh.lua"]:144: in function `StartUpdate'
[string "@BetterBags/data/refresh.lua"]:227: in function `cb'
[string "@BetterBags/core/events.lua"]:49: in function <BetterBags/core/events.lua:47>
[string "=[C]"]: ?
...
[string "@Blizzard_UIParent/Mainline/UIParent.lua"]:422: in function `UIParentLoadAddOn'
[string "@Blizzard_UIParent/Mainline/UIParent.lua"]:627: in function `MajorFactions_LoadUI'
[string "@Blizzard_ExpansionLandingPage/Blizzard_WarWithinLandingPage.lua"]:96: in function `SetUpMajorFactionList'
[string "@Blizzard_ExpansionLandingPage/Blizzard_WarWithinLandingPage.lua"]:89: in function `RefreshMajorFactionList'
[string "@Blizzard_ExpansionLandingPage/Blizzard_WarWithinLandingPage.lua"]:84: in function `RefreshOverlay'
[string "@Blizzard_ExpansionLandingPage/Blizzard_WarWithinLandingPage.lua"]:76: in function <...pansionLandingPage/Blizzard_WarWithinLandingPage.lua:72>
[string "=[C]"]: in function `CreateFrame'
[string "@Blizzard_ExpansionLandingPage/Blizzard_WarWithinLandingPage.lua"]:48: in function `CreateOverlay'
[string "@Blizzard_ExpansionLandingPage/Blizzard_ExpansionLandingPage.lua"]:78: in function `RefreshExpansionOverlay'
[string "@Blizzard_ExpansionLandingPage/Blizzard_ExpansionLandingPage.lua"]:37: in function <...pansionLandingPage/Blizzard_ExpansionLandingPage.lua:35>

Locals:
(*temporary) = "Blizzard_ObjectAPI/Mainline/Item.lua:121: bad argument #1 to 'GetItemGUID' (Usage: local itemGUID = C_Item.GetItemGUID(itemLocation))"
```

**Comment by mLgz0rn** [comment]
Same problem here

```
2x BetterBags/core/async.lua:49: Blizzard_ObjectAPI/Mainline/Item.lua:121: bad argument #1 to 'GetItemGUID' (Usage: local itemGUID = C_Item.GetItemGUID(itemLocation))
[string "=[C]"]: in function `error'
[string "@BetterBags/core/async.lua"]:49: in function <BetterBags/core/async.lua:46>

Locals:
(*temporary) = "Blizzard_ObjectAPI/Mainline/Item.lua:121: bad argument #1 to 'GetItemGUID' (Usage: local itemGUID = C_Item.GetItemGUID(itemLocation))"
```

**Comment by AMaribo** [comment]
On Retail, you'll need to revert to v0.3.7 for now.

**Comment by AMaribo** [comment]
This is fixed in v0.3.10.
